cluster/src/main/java/io/atomix/cluster/impl/PhiAccrualFailureDetector.java

```diff
@@ -22,7 +22,9 @@
 /**
  * Phi Accrual failure detector.
  * <p>
- * Based on a paper titled: "The φ Accrual Failure Detector" by Hayashibara, et al.
+ * A modified version based on a paper titled:
+ * "The φ Accrual Failure Detector" by Hayashibara, et al.
+ * See <a href="https://github.com/atomix/atomix/issues/405">issue 405</a> for more details.
```

Comment:
I don't think linking to the Github issue is necessary.

Reply:
Concur